### ISSUE TITLE
Remove old/stale web front ends

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/fontbakery#1143]
We have had multiple attempts at creating web interfaces for fontbakery:

(1) A long time ago a previous fontbakery developer created an ambitious and complex web panel for monitoring the status of multiple font projects. It used github authentication and it fetched check-results from each individual font repo in order to display them all in a unified table. This required each font project to have a proper fontbakery setup commiting a JSON file with the check results to its own gh-pages branch.

(2) When I started working on Fontbakery, the solution above was somewhat broken and we decided to create a very simple, tiny webview of the check results, to be invoked by the user herself from the command-line with python -m SimpleHTTPServer

(3) More recently I've created a webapp to be deployed on Google App Engine which runs fontbakery on any font file (or set of files) dropped/uploaded to it. And displays the check results on its web ui.

The code for all 3 implementations is currently available on our git repo. (1) is on fontbakery gh-pages branch. (2) and (3) are on master. Solution (2) is located on the root of the repo as an index.html file and a static/ folder, while (3) is under the webapp folder.

I think that (3) is the best solution at the moment from the perspective of user experience and code maintainence. I'd be ok with completely deleting (1) and (2) form our repo.

**Comment by felipesanches** [comment]
We also have the font auto classification tool, but that's an independent (web) tool that has a special purpose of itself. The only thing we may have to take care is not to mix its files with the other webapp files.

**Comment by davelab6** [comment]
> I think that (3) is the best solution at the moment from the perspective of user experience and code maintainence. I'd be ok with completely deleting (1) and (2) form our repo.

And now there is (4) the docker-based dashboard?

I agree that all unmaintained parts should be removed to clean up the repo, and this should be done before tagging the 0.2.0 release. 

I think you should tag the repo (eg `old web ui`)  before deleting them, then it will be easy to find them later if needed.

**Comment by felipesanches** [comment]
there we go:
https://github.com/googlefonts/fontbakery/releases/tag/old_web_ui

**Comment by felipesanches** [comment]
I've just removed (2-simple angular app) to clean up the repo root dir.

Tagging current master won't keep track of (1) which is on gh-pages branch though. But code there does not change at all nowadays. For the record this is the tip of the commit tree on gh-pages branch today: https://github.com/googlefonts/fontbakery/commit/34857fedbf5e416ee7f62046d27074d86786bf38

Then, (3-AppEngine) will probably be merged into (4-Docker).
